### PR TITLE
added-readme and refactored the results page

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Hello everyone I have refactored my results page and also I have added readme Please review this thanks a lot
Live results page-https://deploy-preview-5--piston-quizes.netlify.app/pages/results.html

### CODE REVIEW

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me!

Just one suggestion. Change test result background color. Both red and green are darker shades and also the text is black. Try to make either text or background lighter.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
...
+- [Rules Page](https://piston-quizes.netlify.app/pages/rules.html)
+- [Questions Page](https://piston-quizes.netlify.app/pages/question-pages.html)
+- [Results Page](https://piston-quizes.netlify.app/pages/results.html)
+
+
+---
+
+## 👨‍💻 Connect with me
+
+<a href="https://twitter.com/Akshat86604233"><img src="https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white"/></a>
```
Comment (jalajcodes) [inline README.md:26]: This is fine but just fyi, you can also write this in markdown.
[](https://twitter.com/Akshat86604233)
```md
[![](https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white)](https://twitter.com/Akshat86604233)
```

**Review by jalajcodes — APPROVED** [review]
Looks good to merge!

**Comment by Dev-AkshatGupta** [comment]
> Looks good to me!
> 
> Just one suggestion. Change test result background color. Both red and green are darker shades and also the text is black. Try to make either text or background lighter.

Thanks for reviewing Apurva I will make the changes as suggested

File: README.md (hunk 1 above)
Comment (Dev-AkshatGupta) [reply README.md:26]: Thanks a lot Jalaj for reviewing it I don't know about this But I will study about this and make that in that itself as of now I will pace for the deadline

Comment (jalajcodes) [reply README.md:26]: > Thanks a lot Jalaj for reviewing it I don't know about this But I will study about this and make that in that itself as of now I will pace for the deadline

Sure

**Pull request merged**